src/scripts/components/ResourceIntro.jsx

```diff
@@ -187,7 +187,7 @@ ResourceIntro.propTypes = {
   }),
   children: PropTypes.node,
   title: PropTypes.any,
-  titleSub: PropTypes.string,
+  titleSub: PropTypes.any,
```

Comment:
I asked @marymhart to include this in this PR since we use `Arrays` for rendering tags in in the `ResourceIntro.jsx` component when displaying a list of users